subcommand/server-acl-init/command.go

```diff
@@ -185,6 +188,9 @@ func (c *Command) init() {
 		"Path to file containing ACL token for creating policies and tokens. This token must have 'acl:write' permissions."+
 			"When provided, servers will not be bootstrapped and their policies and tokens will not be updated.")
 
+	c.flags.BoolVar(&c.flagEnableHealthChecksController, "enable-health-checks-controller", false,
```

Comment:
I'm thinking of changing this to something like `add-health-check-rules-to-inject`